src/xdsl/dialects/llvm.py

```diff
@@ -1,11 +1,13 @@
 from __future__ import annotations
 from dataclasses import dataclass
+from typing import TYPE_CHECKING
 
 from xdsl.irdl import (ParameterDef, AnyAttr, irdl_op_builder,
                        irdl_attr_definition, AttributeDef, OperandDef,
                        ResultDef, irdl_op_definition, builder)
-from xdsl.ir import MLContext, ParametrizedAttribute, TYPE_CHECKING, Attribute, Operation
-from xdsl.dialects.builtin import StringAttr, ArrayOfConstraint, ArrayAttr
+from xdsl.ir import (MLContext, ParametrizedAttribute, Attribute, Operation)
+from xdsl.dialects.builtin import (StringAttr, ArrayOfConstraint, ArrayAttr,
+                                   IntegerAttr, IntegerType)
```

Comment:
@webmiche: Are you implementing Stella's suggestion from the other PR, here? I think her suggestion was to use parentheses *and have one package per line*. This makes reading the list itself easier, but also it means that adding and removing a package corresponds to adding and removing exactly one line, so the git diff is easier to read, too.

Maybe you have a different convention in xdsl. In that case, ignore my comment ;)